jenkins-integration/beaker/install/foss/40_clone_test_catalogs.rb

```diff
@@ -0,0 +1,16 @@
+test_name 'Clone test-catalogs repo onto master'
+
+# This is awful - we clone locally and then SCP the whole thing over.
+# See QENG-2556 for adding a proper way to do this in Beaker.
+tmp_repo = "/tmp/test-catalogs-#{Process.pid}"
```

Comment:
Yeah Beaker only helps with that on the host nodes, not the local coordinator machine.

I actually don't want to make the directory, because the git clone will complain if it already exists.